web/src/engine/osk/src/views/oskView.ts

```diff
@@ -252,22 +251,10 @@ export default abstract class OSKView
       this.uiStyleSheetManager.linkExternalSheet(sheetHref);
     }
 
-    this.activeKeyboard = this.config.keyboardToActivate;
-
-    // Ensure the appropriate banner mode is activated; it's possible for a model to have either
-    // partially or fully loaded, especially if the prior line has a full Keyboard instance.
-    const modelState = this.config.predictionContextManager?.modelState;
-    if(modelState) {
-      this.bannerController.selectBanner(modelState);
-    }
-
     this.setBaseMouseEventListeners();
     if(this.hostDevice.touchable) {
       this.setBaseTouchEventListeners();
     }
-
-    this._Box.style.display = 'none';
```

Comment:
Given the context of the comment on #12015, I believe that this specific line should not be reverted.

Reply:
Given further discussion there, it sounds like we'll go ahead and revert it for now, with a focused... "unreversion"?... of this as a quick followup.